packages/application-shell/src/components/navbar/navbar.tsx

```diff
@@ -263,6 +265,10 @@ const NavBar = (props: TNavbarProps) => {
     [props.project]
   );
 
+  if (props.isLoading) {
+    return <NavBarSkeleton isExpanded={isMenuOpen} />;
+  }
```

Comment:
Unlike previously, as we have 2 states of the skeleton (expanded and non-expanded), and the new navbar skeleton is rendered inside the navbar